src/sonic-yang-models/tests/files/sample_config_db.json

```diff
@@ -1290,6 +1290,56 @@
                 "login": "local"
             }
         },
+        "SYSTEM_PORT": {
```

Comment:
kindly point to the source of truth for config, e.g. HLD, Orchagent processing functions etc. Thx

Reply:
Please see `PortsOrch::addSystemPorts()` in sonic-swss/orchagent/portsorch.cpp